### ISSUE TITLE
very fast inference, so far one of the best

### ISSUE THREAD

**Issue by Arvrairobo** [X-LANCE/AniTalker#39]
i have successfully installed and able to produce the video, i have noticed two things:

1) when image has a beards then it is not able to generate proper output, infact it is worst in that case.
2) if image is not a close up or face is far in the photo or image is a full body photo then it is not able to create talking face, instead the behaviour should be, it should crop the face, make it animated and paste back or similar.
3) lipsync sometimes poort, sometimes good but still need improvement.

what other languages are supported?

Any update on the next release? eagarly waiting for it.

**Comment by Arvrairobo** [comment]
please let me know if any update on this? as we need those features to work, any one can help?

**Comment by nitinmukesh** [comment]
@Arvrairobo 

regarding second point, please try the new WebUI and post results.
https://github.com/X-LANCE/AniTalker/issues/40

**Comment by Arvrairobo** [comment]
@nitinmukesh yes perfect, it has worked for me by following to your tutorial as well as your repo, but i am finding other challenges which i have written in https://github.com/X-LANCE/AniTalker/issues/40
